error[E0433]: failed to resolve: could not find `arch` in `self`
  --> src/lib.rs:64:37
   |
64 |         let result = unsafe { self::arch::__cpuid_count(a, c) };
   |                                     ^^^^ could not find `arch` in `self`

error: could not compile `raw-cpuid` (lib) due to previous error; 3 warnings emitted

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -53,7 +53,7 @@ extern crate bitflags;
 pub mod native_cpuid {
     use super::CpuIdResult;
 
-    #[cfg(all(target_arch = "x86", not(target_env = "sgx"), target_feature = "sse"))]
+    #[cfg(all(target_arch = "x86", not(target_env = "sgx"), not(target_feature = "sse")))]
     use core::arch::x86 as arch;
     #[cfg(all(target_arch = "x86_64", not(target_env = "sgx")))]
     use core::arch::x86_64 as arch;
